doc/application_workflow/private_registry.md

```diff
@@ -0,0 +1,102 @@
+INFO[2016-10-08T17:31:06-07:00] docker -H 192.168.218.138:2375 info          
+INFO[2016-10-08T17:31:06-07:00] Installer completed successfully             
+
+$> DOCKER_HOST=tcp://192.168.218.138:2375 DOCKER_API_VERSION=1.23 docker run -d -p 5000:5000 --name registry registry:2
+
+$> ./vic-machine-linux create --target=192.168.218.207 --image-store=datastore1 --name=vic-app --user=root --password=vagrant --compute-resource="/ha-datacenter/host/esxbox.localdomain/Resources" --bridge-network=vic-network --no-tls --volume-store=datastore1/vic-app:default --force --docker-insecure-registry 192.168.218.138
+
+...
+INFO[2016-10-08T17:31:06-07:00] Initialization of appliance successful       
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00] vic-admin portal:                            
+INFO[2016-10-08T17:31:06-07:00] http://192.168.218.131:2378                  
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00] Docker environment variables:                
+INFO[2016-10-08T17:31:06-07:00]   DOCKER_HOST=192.168.218.131:2375           
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00]                                              
+INFO[2016-10-08T17:31:06-07:00] Connect to docker:                           
+INFO[2016-10-08T17:31:06-07:00] docker -H 192.168.218.131:2375 info          
+INFO[2016-10-08T17:31:06-07:00] Installer completed successfully             
+
+$> sudo vi /lib/systemd/system/docker.service
+$> sudo systemctl daemon-reload
+$> sudo systemctl restart docker
+$> docker tag busybox 192.168.218.138:5000/test/busybox
+$> docker push 192.168.218.138:5000/test/busybox
+```
+
+terminal 2:
+```
+$> export DOCKER_HOST=tcp://192.168.218.131:2375
+$> export DOCKER_API_VERSION=1.23
+$> docker pull 192.168.218.138:5000/test/busybox
+```
+
+Note, in this example, we disabled TLS for simplicity.  Also, we did not show what was modified in /lib/systemd/system/docker.service.  That is shown below for the example above.
+
+```
+[Service]
+Type=notify
+# the default is not to use systemd for cgroups because the delegate issues still
+# exists and systemd currently does not support the cgroup feature set required
+# for containers run by docker
+ExecStart=/usr/bin/dockerd --tls=false -H fd:// --insecure-registry 192.168.218.138:5000
+```
+
+Also, note in terminal 1 above, in the second step where we run the Docker registry.  In that command, we specify the necessary environment variables before the docker run command.  On our Linux machine, this sets the variables for the duration of the operation, and once the docker run finishes, those variables are reverted back to their previous values.  Also, note that we run registry:2.  It is important not to specify registry:2.0 in this example.  Registry 2.0 had issues that prevents the example above from working.
```

Comment:
Please reword this to something like

`In the second step for terminal 1 above, we specify...`